scripts/preprocessing/02-frequency_filter.py

```diff
@@ -72,7 +72,15 @@ def run_filter(subject, run=None, session=None):
     logger.info(gen_log_message(message=msg, step=2, subject=subject,
                                 session=session, run=run,))
 
-    raw = mne.io.read_raw_fif(raw_fname_in)
+    read_raw = mne.io.read_raw_fif
```

Comment:
Hm, we do this nowhere else in the pipeline, do you feel strongly about keeping this?